tests/FirebaseProjectManagerTest.php

```diff
@@ -52,8 +52,8 @@ public function a_default_project_can_be_set(): void
 
         $manager->setDefaultProject($projectName);
 
-        $this->assertEquals($projectName, $manager->getDefaultProject());
-        $this->assertEquals($projectName, $this->app->config->get('firebase.default'), 'default project should be set in config');
+        self::assertSame($projectName, $manager->getDefaultProject());
```

Comment:
There's no added value in changing this, but I'm willing to merge it if you replace `self::` with `$this->` 😅

Reply:
@jeromegamez done